include/icubFloatingBaseEstimatorV1.h

```diff
@@ -0,0 +1,632 @@
+/*
+################################################################################
+#                                                                              #
+# Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia (IIT)          #
+# All Rights Reserved.                                                         #
+#                                                                              #
+################################################################################
+
+# @authors: Prashanth Ramadoss <prashanth.ramadoss@iit.it>
+#           Giulio Romualdi    <giulio.romualdi@iit.it>
+#           Silvio Traversaro  <silvio.traversaro@iit.it>
+#           Daniele Pucci      <daniele.pucci@iit.it>
+*/
+
+#ifndef ICUB_FLOATING_BASE_ESTIMATOR_V1_H
+#define ICUB_FLOATING_BASE_ESTIMATOR_V1_H
+
+#include <iDynTree/Estimation/AttitudeMahonyFilter.h>
+#include <iDynTree/Estimation/SimpleLeggedOdometry.h>
+#include <iDynTree/Estimation/BipedFootContactClassifier.h>
+#include <iDynTree/Estimation/AttitudeQuaternionEKF.h>
+
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/KinDynComputations.h>
+#include <iDynTree/ModelIO/ModelLoader.h>
+#include <iDynTree/yarp/YARPConversions.h>
+#include <iDynTree/Core//EigenHelpers.h>
```

Comment:
```suggestion
#include <iDynTree/Core/EigenHelpers.h>
```

Reply:
Fixed in https://github.com/robotology/whole-body-estimators/pull/1/commits/714c6ccc684abcde00050378b2ddb09d6054ddd9